docs/End_to_End_Testing.md

```diff
@@ -0,0 +1,70 @@
+# End-to-End Testing

```

Comment:
Here we have some formatting issues.

I think the best is to use a Markdown preview 👍